### PR TITLE
Update name for RegionUserCount in APIBacklog.md

### LINKED ISSUE #48: API name change - from RegionUserCount to RegionDeviceCount
**Problem description**
The naming of the Region User Count project does not match the actual functionality of the API

**Possible evolution**
After discussion in RUC work group, we believe that switch the API name from RegionUserCount to RegionDeviceCount is more accurate because operators can count the number of connected devices rather than the number of users. Specific discussions can refer to https://github.com/camaraproject/RegionUserCount/issues/8

**Alternative solution**
Maybe RegionSimCount is more accurate,but I think it is too technical and not easy to understand

PR is [here](https://github.com/camaraproject/APIBacklog/pull/47)

### PR DESCRIPTION
**Pull request by TEF-RicardoSerr**
#### What type of PR is this?

Add one of the following kinds:
* correction

#### What this PR does / why we need it:
Update of Region User COunt name 

#### Which issue(s) this PR fixes:

Fixes #48

### CODE REVIEW

**Review by hdamker — APPROVED** [review]

**Comment by eric-murray** [comment]
@camaraproject/api-backlog_codeowners
This needs to be approved by a codeowner before it can be merged

**Comment by hdamker** [comment]
@caubut-charter would you shortly approve? The change repository name change is done already months ago.

**Review by caubut-charter — APPROVED** [review]
Change looks good.

**Pull request merged**
